tests/scripts/app_command.lua

```diff
@@ -54,6 +54,53 @@ do -- ExportSpriteSheet
     11,8,11,21,11,8,11,
     11,8,11,21,11,11,8,
   })
+
+  local s = Sprite{ fromFile="sprites/4f-index-4x4.aseprite" }
+  app.command.ExportSpriteSheet {
+    type=SpriteSheetType.PACKED,
+    textureFilename="_test_export_spritesheet1.png",
```

Comment:
I think we should use a different name here? There is another test that uses this filename already. Maybe _test_export_spritesheet2.png?